spec/paper_trail_spec.rb

```diff
@@ -94,37 +94,8 @@
   end
 
   describe ".version" do
-    it { expect(described_class).to respond_to(:version) }
```

Comment:
I'd probably prefer to have some delegation/deprecation tests, e.g. to test deprecation issues such as changing the method signature to `controller_info`.

This could be done tersely with a shared example if you want to write less code:
```ruby
shared_example 'it delegates to request' do |method, *args, &block|
  it do
    expect(PaperTrail::Request).to receive(:method).with(args, &block).and_return(:some_return_value)
    expect(described_class.send(:method, args, &block).to_return(:some_return_value)
 end
end
```
```ruby
it_behaves_like 'it delegates to request', :controller_info, [], nil
it_behaves_like 'it delegates to request', :whodunnit, [:some_whodunnit] { 'some_block' }
```

Reply:
Not sure if the above will work exactly as is, but that's the idea.  Could be just as straightforward to do it more explicitly for each method, but I think it would be less error-prone this way.